config/controller/deployment.yaml

```diff
@@ -21,7 +21,6 @@ spec:
           imagePullPolicy: IfNotPresent
           args:
             - crd-apply
-      terminationGracePeriodSeconds: 10
```

Comment:
Just curious why this changed

Reply:
It's actually a duplicate, you can see the same setting on line 42. Somehow it's never been a problem before, but you can see a test failure in the first commit:
https://app.circleci.com/pipelines/github/dominodatalab/forge/563/workflows/989d07ec-5e07-4688-bb0b-500ccadd328c/jobs/1040

Reply:
Oh because K3s isn't strictly versioned, so it seems 1.21 is more strict.